### PR TITLE
Remove opt-in path form all links in digest templates

### PR DESCRIPTION
**Pull request by phamann**
Their is a B2B requirement to remove all forced opt-in links from the myFT digest emails to ensure the users current explicit opt-in status is maintained.

Horrible PR to review but does clean up our links as no longer have to encode for the query param.

### CODE REVIEW

**Comment by keirog** [comment]
`%3A` = `:`, not `-` but otherwise 👍

File: functions/create-email/views/partials/footer-plain.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 
 Unsubscribe
-https://next.ft.com/__opt-in?optedvia=myft-email&referrer=https%3A%2F%2Fnext.ft.com/login?location=https%3A%2F%2Fnext.ft.com%2Fmyft%2Femail%2Funsubscribe%2F{{uuid}}
+http://next.ft.com/login?location=https%3A%2F%2Fnext.ft.com%2Fmyft%2Femail%2Funsubscribe%2F{{uuid}}
```
Comment (keirog) [inline functions/create-email/views/partials/footer-plain.html]: Should this be http?

Comment (debugwand) [inline functions/create-email/views/partials/footer-plain.html]: not https? it initially takes you to login

Comment (debugwand) [inline functions/create-email/views/partials/footer-plain.html]: will result in a sorry page if user is not opted in and tries to unsubscribe.

File: functions/create-email/views/partials/concept-url-plain.html [hunk 2]
```diff
@@ -1 +1 @@
-https://next.ft.com/__opt-in?optedvia=myft-email&referrer=https%3A%2F%2Fnext.ft.com%2Fstream%2F{{{taxonomy}}}Id%2F{{{uuid}}}&hash=myft%3Anotification%3A{{{@root.type}}}-email%3Atag
\ No newline at end of file
+https://next.ft.com/stream/{{{taxonomy}}}Id/{{{uuid}}}#myft:notification:{{{@root.type}}}-email:tag
```
Comment (keirog) [inline functions/create-email/views/partials/concept-url-plain.html]: The url partials can’t have trailing newlines (concept-url-plain.html, concept-url.html, content-url-plain.html, content-url.html)

**Pull request merged**